crates/re_space_view_spatial/src/contexts/transform_context.rs

```diff
@@ -321,6 +318,27 @@ fn get_cached_transform(
     transform3d
 }
 
+fn get_cached_pinhole(
```

Comment:
Why not change the implementation of `query_pinole` to use the cache instead?

Reply:
ripples too much right now and didn't want to have that in this PR.
Actually many callsites of `query_pinhole` only want to know about whether the component exist, so they shouldn't do the full query (go to `EntityTree` instead!)